pkg/vulnsrc/vulnsrc.go

```diff
@@ -136,60 +136,83 @@ type Optimizer interface {
 }
 
 type fullOptimizer struct {
-	dbc db.Operation
+	dbOp db.Operation
```

Comment:
This change makes this PR bigger. I think we should revert this change now. If you actually want to change this name, we can do it in another PR. But if there is a strong reason, we can keep it as is in this PR.
I'm guessing this is because you added and removed `dbConfig`.

Reply:
OK I've changed it.